### PR TITLE
Update stateToParams.js

### PR DESCRIPTION
**Pull request by Zveroboev**
Fix boolean case

### CODE REVIEW

**Comment by Zveroboev** [comment]
'currentItemState' can be boolean. Particularly false

File: src/stateToParams.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ export function stateToParams(initialState, currentState, location) {
       isDefault = typeof(currentItemState) === 'object' ? isEqual(initialValue, currentItemState) : currentItemState === initialValue;
     }
     // if it is default or doesn't exist don't make a query parameter
-    if (((!currentItemState && !options.serialize) || isDefault) && !options.setAsEmptyItem) {
+    if (((typeof(currentItemState) === undefined && !options.serialize) || isDefault) && !options.setAsEmptyItem) {
```
Comment (joshbDev) [inline src/stateToParams.js]: I tried this in the console and I think the syntax is wrong here. I think it should be `typeof currentItemState === "undefined" && ...`
Screen Shot 2020-01-21 at 1 56 22 PM

Comment (Zveroboev) [reply src/stateToParams.js]: Oh, yes. Of course. "undefined"

Comment (Zveroboev) [reply src/stateToParams.js]: with brackets -> 
image

:)

**Pull request closed without merging**